src/snowflake/cli/api/console/abc.py

```diff
@@ -68,6 +68,7 @@ def phase(
         self,
         enter_message: str,
         exit_message: Optional[str] = None,
+        span_name: Optional[str] = None,
```

Comment:
not sure if this is truly needed, since we can use multiple context managers on the same line. Thoughts?

Reply:
I forgot about that feature, I adjusted these instances, thanks for the reminder